vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/view/components/ProjectDetailsComponent.java

```diff
@@ -149,12 +201,26 @@ private void loadProjectData(Project project) {
       this.selectedProject = project.getId();
       component.titleField.setValue(project.getProjectIntent().projectTitle().title());
       component.projectObjective.setValue(project.getProjectIntent().objective().value());
+
       project.getProjectIntent().experimentalDesign().ifPresentOrElse(
           experimentalDesignDescription -> component.experimentalDesignField.setValue(
               experimentalDesignDescription.value()),
-          () -> component.experimentalDesignField.setPlaceholder("No description yet."));
+          () -> component.experimentalDesignField.setValue(""));
+
+
+      project.getProjectManager().ifPresentOrElse(it -> {
+        component.pmElement.setContent(it.fullName(),it.getEmailAddress());
+        component.projectManagerComboBox.setValue(it);
+      }, component.pmElement::reset);
```

Comment:
what do you think of renaming the method to `clear` as the content is cleared. (similar to text fields)